src/CONST.js

```diff
@@ -175,6 +175,7 @@ const CONST = {
     DATE: {
         MOMENT_FORMAT_STRING: 'YYYY-MM-DD',
         SQL_DATE_TIME: 'YYYY-MM-DD HH:mm:ss',
+        FNS_FORMAT_STRING: 'yyyy-MM-dd',
```

Comment:
Why is this different than the moment format? @waterim

Reply:
Because moment is using its own formatting, and date-fns using ECMAScript Internationalization API, thats why its different

Reply:
@mountiny

Reply:
Thanks for clarifying 🙇